JSDemos/Demos/DataGrid/ExcelJSHeaderAndFooter/description.md

```diff
@@ -1,8 +1,8 @@
-<a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> library allows you to customize worksheets outside of the exported cell area. This demo uses this functionality to add a header (a title before exported data) and a footer (a note after exported data).  
+<a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> library allows you to customize worksheets outside of the exported cell area. This demo uses this functionality to add a header (a title before exported data) and a footer (a note after exported data).
 
-Review the [onExporting](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#onExporting) handler to see the data export code. The demo illustrates the following customization features:     
+Review the [onExporting](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#onExporting) handler to see the data export code. The demo illustrates the following customization features:
```

Comment:
I'd be a bit more specific here. The demo is about header/footer, as we already mentioned. So now we need to narrow it down to just the methods that customize header and footer, not the entire demo. 

```suggestion
Review the [onExporting](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#onExporting) handler to see the data export code. The functions that create header and footer sections utilize the following customization features:
```